### ISSUE TITLE
basic setup fails on error "a branch named 'master' already exists"

### ISSUE THREAD

**Issue by spacewaffle** [robotology/gh-action-nightly-merge#4]
This is what my yaml file looks like. Notice checkout was bumped to v2. Not sure if that should change anything. Action is failing on the checkout which seems to make sense. Can't checkout a new branch called master if it already exists locally.

```
git fetch origin master
git checkout -b master origin/master
```

```
name: 'Auto Merge Master to Staging'

on:
  push:
    branches:
      - master

jobs:
  auto-merge-master-to-staging:

    runs-on: ubuntu-latest

    steps:
    - name: Checkout
      uses: actions/checkout@v2

    - name: Merging Master Into Staging
      uses: robotology/gh-action-nightly-merge@v1.2.0
      with:
        development_branch: 'staging'
        allow_ff: false
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

**Comment by spacewaffle** [comment]
Just confirmed the action works with checkout@v1. It's incompatible with checkout@v2. Ya'll should consider supporting v2 since the official github docs are pointing to using v2 over v1 now.

**Comment by pruthvi-rollick** [comment]
Indeed works with checkout@v1 but if we were to attempt a second merge to a different development branch, it fails again with the same error.
![image](https://user-images.githubusercontent.com/54266310/86365044-9cccb880-bc96-11ea-8ee4-1e0c6443de35.png)

**Comment by Toilal** [comment]
I wrote a pull request that should fix this issue.

**Comment by drdanz** [comment]
Fixed by #5 and released in v1.3.2
